main.py

```diff
@@ -27,6 +27,13 @@ def UpdateQueue():
         api.request('statuses/retweet/:' + str(post))
         post_list.pop(0)
 
+def CheckForFollowRequest(item):
+    text = item['text']
+    if "Follow" in text:
```

Comment:
Good call on lowercase follow. Hardly ever use python, didn't even think about case sensitivity. Never used word boundaries before. That just to catch people doing stupid stuff with the word "follow"?